app/src/components/main/modal/modalLayout/modalLayout.scss

```diff
@@ -20,7 +20,7 @@
   display: block;
   width: 100%;
   height: 100%;
-  background-color: #000000;
+  background-color: #000000; /* stylelint-disable-line color-no-hex */
```

Comment:
Why not short format like `#000`?

Reply:
The most important question it why there is a color which is not in the palette.  All lines marked with `stylelint-disable-line` will be investigated in the future.